LordUche/ireporter-react
# #164034245 Add Report Incident Functionality

#### What does this PR do?
Adds report incident functionality
#### Description of Task to be completed?
- create report incident form component
- setup reducer and action creator
- connect the component to the redux store
- write tests
#### How should this be manually tested?
- ensure that the build is passing on Travis CI
- check the [review app](https://uche-ireporter-react-stag-pr-4.herokuapp.com/) to make sure that all is well
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#164034245](https://www.pivotaltracker.com/story/show/164034245)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

[Finishes #164034245]
